### DISCUSSION TITLE
Add weather update time in desktop widgets

### DISCUSSION THREAD

**Discussion by KERWINWANG510** [breezy-weather/breezy-weather#1166] [Ideas]
### Describe your idea

May I ask if it is possible to add a weather update time display function in the desktop widget, as shown in the picture below, so that I can know the last update time and confirm whether the displayed weather is valid? At the same time, when I click the refresh button, I can update the weather without opening the app。
![微信图片_20240701130559](https://github.com/breezy-weather/breezy-weather/assets/30833200/19ecedf4-1de1-4146-8aa1-ca1e3f8f1573)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Yes, you can add refresh time in widgets, it already exists. It's the default configuration for "Subtitle data".
